ignite-base/android/app/build.gradle

```diff
@@ -78,12 +84,12 @@ def enableProguardInReleaseBuilds = false
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    buildToolsVersion "23.0.1"
```

Comment:
Not a huge deal, but 23.0.3 is the latest in that branch.   I'm guessing you didn't change these manually though.